src/Header/Header.js

```diff
@@ -1,14 +1,24 @@
+import { useRef, useState } from 'react'
+
 import Logo from './Logo'
 import SocialMedia from '../Integrations/SocialMedia'
 import Ham from './Hamburger/Hamburger'
 import HamMenu from './Hamburger/HamMenu'
 
+import { useOnClickOutside as closeHeader} from './Hamburger/hooks';
+
 import { Business_Hours, Contact_Us, Location, Header, LeftSide_Header, RightSide_Header } from './Header.styled'
 
 
-export default function HeaderArea({ node, scroll , setOpen , isOpen , closeMenu }) {
-    const location = "https://www.google.com/maps/place/4155+University+Pkwy,+San+Bernardino,+CA+92407/@34.1660689,-117.3342482,17z/data=!3m1!4b1!4m5!3m4!1s0x80c352273252f45b:0x3de888116e7c46bf!8m2!3d34.1660689!4d-117.3320595"
+const location = "https://www.google.com/maps/place/4155+University+Pkwy,+San+Bernardino,+CA+92407/@34.1660689,-117.3342482,17z/data=!3m1!4b1!4m5!3m4!1s0x80c352273252f45b:0x3de888116e7c46bf!8m2!3d34.1660689!4d-117.3320595"
+export default function HeaderArea({ scroll }) {
     
+//handles open close of hamburger
+  const node = useRef();
+  const [isOpen, setOpen] = useState(false)
+  const closeMenu = () => {setOpen(false);}
+  closeHeader(node, () => setOpen(false));
```

Comment:
What is this doing?

Reply:
That attaches a reference node to the header - close header is being imported from the hooks for the on click outside. When a user clicks outside of the header/hamburger, it'll close the hamburger. I imported it as closeHeader because the same thing is used for the modal, which I imported as closeModal.